### ISSUE TITLE
readlines: undefined method `readlines' for #<File:0x220>

### ISSUE THREAD

**Issue by cmoulliard** [asciidoctor/asciidoctor.js#208]
The following code 

```
    var f = path.join('test', 'fixtures', 'simple.adoc');
    console.log("File %s",f);
    var expected = getFile(path.join('test', 'fixtures', 'simple.html')).contents.toString('utf8');

    var options = opal.hash({doctype: 'article',
        header_footer: 'true',
        safe: 'unsafe',
        attributes: ['nofooter']});

    var result = opal.Asciidoctor.$convert_file(f, options); // Line 86
```

generates this error 

```
readlines: undefined method `readlines' for #<File:0x220>
    at Opal.defs.TMP_1 [as $new] (/Users/chmoulli/MyProjects/asciidoctor-nodejs/node_modules/opal-runtime/src/opal.js:4853:15)
    at $File.ːmethod_missing (/Users/chmoulli/MyProjects/asciidoctor-nodejs/node_modules/opal-runtime/src/opal.js:3410:54)
    at $File.method_missing_stub [as $readlines] (/Users/chmoulli/MyProjects/asciidoctor-nodejs/node_modules/opal-runtime/src/opal.js:1006:35)
    at module_constructor.ːload (/Users/chmoulli/MyProjects/asciidoctor-nodejs/node_modules/asciidoctor.js/dist/npm/asciidoctor-core.js:12299:27)
    at module_constructor.ːconvert (/Users/chmoulli/MyProjects/asciidoctor-nodejs/node_modules/asciidoctor.js/dist/npm/asciidoctor-core.js:12410:20)
    at module_constructor.ːconvert_file (/Users/chmoulli/MyProjects/asciidoctor-nodejs/node_modules/asciidoctor.js/dist/npm/asciidoctor-core.js:12516:21)
    at Test.<anonymous> (/Users/chmoulli/MyProjects/asciidoctor-nodejs/test/render.js:86:35)

```

**Comment by Mogztter** [comment]
I think this is fixed in https://github.com/asciidoctor/asciidoctor.js/pull/199
Could you please give it a try ?

**Comment by cmoulliard** [comment]
Test done with the fix and all the tests are green here : https://github.com/cmoulliard/asciidoctor-nodejs/blob/master/test/render.js

**Comment by Mogztter** [comment]
Resolved by #199
